scripts/buildExamples.fish

```diff
@@ -47,9 +47,21 @@ and echo "Generating swagger"
 and rm -rf js/apps/system/_admin/aardvark/APP/api-docs.json
 and bash -c ./utils/generateSwagger.sh
 and bash -c "cd Documentation/Scripts && python ./codeBlockReader.py"
+and begin
+  if test -f ./utils/generateAllMetricsDocumentation.py
```

Comment:
This only tests whether the script exists, right?
Should also run the script without `-d` for validation purposes!

Reply:
Done.